bears/python/requirements/PySafetyBear.py

```diff
@@ -5,7 +5,8 @@
 from safety import safety
```

Comment:
It has to do something with lines like these in tests:
```python
with mock.patch(
            'bears.python.requirements.PySafetyBear.safety.check',
            return_value=[],
        ) as check:
```
when I remove `from safety import `safety` in the Bear. All test fail.